.kitchen.yml

```diff
@@ -31,7 +31,7 @@ platforms:
       box: windows-8.1
       box_url: http://cdn.box-cutter.com/windows/virtualbox4.3.20/eval-win81x64-enterprise-nocm-1.0.2.box
     provisioner:
-      require_chef_omnibus: 12.0.0
+      require_chef_omnibus: 12
```

Comment:
I am a PS newb - can we look at that together?  Or would you rather I file a separate issue?